portal-ui/src/screens/Console/Tenants/TenantDetails/TenantMonitoring.tsx

```diff
@@ -0,0 +1,324 @@
+// This file is part of MinIO Console Server
+// Copyright (c) 2021 MinIO, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import React, { Fragment, useEffect, useState } from "react";
+import { connect } from "react-redux";
+import { Theme } from "@mui/material/styles";
+import createStyles from "@mui/styles/createStyles";
+import withStyles from "@mui/styles/withStyles";
+import Grid from "@mui/material/Grid";
+import {
+  actionsTray,
+  containerForHeader,
+  searchField,
+  tenantDetailsStyles,
+} from "../../Common/FormComponents/common/styleLibrary";
+import { Button, CircularProgress, Divider } from "@mui/material";
+import Paper from "@mui/material/Paper";
+import { ITenant } from "../ListTenants/types";
+import { setErrorSnackMessage } from "../../../../actions";
+import { ErrorResponseHandler } from "../../../../common/types";
+import EditTenantMonitoringModal from "./EditTenantMonitoringModal";
+
+import api from "../../../../common/api";
+import { EditIcon } from "../../../../icons";
+import FormSwitchWrapper from "../../Common/FormComponents/FormSwitchWrapper/FormSwitchWrapper";
+import { ITenantMonitoringStruct } from "../ListTenants/types";
+import { IKeyValue } from "../ListTenants/types";
+import KeyPairView from "./KeyPairView";
+import ConfirmationDialog from "./ConfirmationDialog";
```

Comment:
Please rebase with master and use `ConfirmDialog` for ux consistency 
```suggestion
import ConfirmDialog from "../../Common/ModalWrapper/ConfirmDialog";
```